index.js

```diff
@@ -182,24 +189,44 @@ Liftoff.prototype.launch = function (opts, fn) {
     return this.completions(completion);
   }
 
+  opts = mapValues(opts);
+
+  var getEnvironment;
+  if (process.env.__FLAGGED_RESPAWNED) {
+    opts.argv = process.argv;
+    getEnvironment = function(opts) {
+      return this.buildEnvironment(opts);
+    }.bind(this);
+  } else {
+    var env = this.buildEnvironment(opts);
+    getEnvironment = function() {
+      return env;
+    };
+  }
+
   this.handleFlags(function (err, flags) {
     if (err) {
       throw err;
     } else {
       if (flags) {
-        flaggedRespawn(flags, process.argv, function (ready, child) {
+        if (process.env.__FLAGGED_RESPAWNED) {
```

Comment:
I'd like to avoid this, can we change the implementation to avoid it?

Reply:
I have two ideas to avoid this:

1. Adds a new cli option `--ignore-v8flags` not to flagged-respawn, and liftoff always adds this flag when flagged-respawning.

2. Uses `child_process.fork` instead of `child_process.spawn` and if `process.connected` is true, liftoff always doesn't fork. (It is needed to create a new module using fork like `flagged-respawn`).